test/actions/profile.spec.js

```diff
@@ -20,7 +20,7 @@ describe('addComment', () => {
     expect(typeof addComment).toBe('function');
   });
 
-  it('should create an appropiate action', () => {
+  it('should create an appropriate action', () => {
```

Comment:
lets get simple in our wording 
```suggestion
  it('should create an action', () => {
```